submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/submit/FileListValidationTest.kt

```diff
@@ -255,10 +268,27 @@ class FileListValidationTest(
         webClient.deleteFile(fileList.name)
     }
 
+    @Test
+    fun `empty file list on behalf another user`() {
```

Comment:
personally I think this combination of test is not good. I mean in ideal world when time and budget is not a constrain will be great to have combination test of all our features like multipart requesst on behalf or behalf submission with root path but as we are not there introducing some combination is not good unless you see particual value in this one. IMO https://github.com/EBIBioStudies/biostudies-backend-services/pull/644/files#diff-35b0dc1882619e1c82a0f167ba73be7b1521e0af195c7d095a8cfc70a74d01c5R251 is enough.